libs/coin-evm/src/api/etherscan.ts

```diff
@@ -102,7 +102,28 @@ export const getLastTokenOperations = async (
     url,
   });
 
-  return ops.map((event, index) => etherscanERC20EventToOperations(accountId, event, index)).flat();
+  // Why this thing ?
+  // Multiple events can be fired by the same transactions and
+  // those transfer events can go from anyone to anyone, which
+  // means that multiple events could be sent to or from the
+  // same address during the same transaction.
+  //
+  // To make sure every event (transformed into an operation here)
+  // has a unique id, we're groupping them by transaction hash
```

Comment:
```suggestion
  // has a unique id, we're grouping them by transaction hash
```

Reply:
Like said to Hedi, I'll do a last update/rebase once it's on the main branch, I can't stand the idea of doing yet another rebase, that will force me to rebase also the UI PR just for 1 letter haha